Tasks/VsTest/taskinputparser.ts

```diff
@@ -117,6 +117,11 @@ function initTestConfigurations(testConfiguration: models.TestConfigurations) {
     testConfiguration.tiaConfig = getTiaConfiguration();
 
     testConfiguration.pathtoCustomTestAdapters = tl.getInput('pathtoCustomTestAdapters');
+    if (testConfiguration.pathtoCustomTestAdapters &&
+        !utils.Helper.pathExistsAsDirectory(testConfiguration.pathtoCustomTestAdapters)) {
```

Comment:
what's the old behavior? is it going to cause build breaks because of this change?